bleak/backends/winrt/client.py

```diff
@@ -258,14 +258,18 @@ def _ConnectionStatusChanged_Handler(sender, args):
 
         return True
 
-    async def disconnect(self) -> bool:
+    async def disconnect(self, **kwargs) -> bool:
         """Disconnect from the specified GATT server.
 
+        Keyword Args:
+            timeout (float): Defaults to 10.0.
```

Comment:
Perhaps specify the unit as seconds; so "timeout (float): Defaults to 10.0 seconds."